### ISSUE TITLE
[BUG] Minifb example crashes on Windows (while transmutting buffer)

### ISSUE THREAD

**Issue by SimplyNaOH** [38/plotters#180]
**Describe the bug**
Runing minifb example (Windows x64; same library versions as in [the example's Cargo.toml](https://github.com/38/plotters/blob/b0606484896a3b77f75c3e52ea889044f7515f9e/examples/minifb-demo/Cargo.toml); either debug or release compilation) I get this error: ```error: process didn't exit successfully: `target\release\test.exe` (exit code: 0xc0000005, STATUS_ACCESS_VIOLATION)```.

I tracked down the error to this line:
https://github.com/38/plotters/blob/b0606484896a3b77f75c3e52ea889044f7515f9e/examples/minifb-demo/src/main.rs#L137

I am new to rust, so the only way I could solve this is by inefficiently copying the data to a new buffer:
```rust
let mut buf2 = vec![0u32; W * H];
for i in (0..buf.len()).step_by(4) {
    buf2[i / 4] = u32::from_ne_bytes(buf[i..i+4].try_into().unwrap());
};
window.update_with_buffer(&buf2);
```

The above solves the issue but leaves me wondering whether there a way to do this more efficiently? and also, did the original example code work under some other circumstances? how so?

**Version Information**
```toml
minifb = "0.13.0"
plotters = {version = "^0.3.0", default_features = false, features = ["ttf"]}
plotters-bitmap = {version = "^0.3.*", default_features = false}
```

**Comment by 38** [comment]
Hi there, thanks for reporting this. This is a bug of the example, which transmute the buffer mistakenly (with the wrong size) - This is supposed to be fixed by commit ebc2307.

The detailed explain is `std::mem::transmute` doesn't handles the slice size, so this gives the transmuted slice a wrong size. 

Unfortunately, I can't confirm this works on Windows since I don't have a Windows machine. It would be very helpful if you can confirm this actually fix the issue!

**Comment by SimplyNaOH** [comment]
ebc2307 does fix the issue in Windows as well; thanks!
